api/swagger.yaml

```
@@ -167,7 +212,8 @@ paths:
                       properties:
                         exposedPort:
                           description: The port inside the container.
-                          type: uint
+                          type: integer
+                          minimum: 0
```

```suggestion
                          minimum: 0
                          maximum: 65535
```

We might also specify the maximum port number but don't need to.